Automattic/wp-calypso
# Concierge session dashboard

#### Changes proposed in this Pull Request

**NOTE: Depends on D30911-code and D30912-code.**

* Implements an appointment dashboard that shows details for the upcoming concierge appointment.
* After a user completes booking a session, navigating to `/me/concierge/{SITE_SLUG}/book` will show the appointment details, including a button to reschedule or cancel the session.

Screenshot:

![calypso localhost_3000_me_concierge_qz954002103 wordpress com_book](https://user-images.githubusercontent.com/1269602/62519359-06ef9a00-b849-11e9-9e5b-29ece18933e7.png)

When the Zoom session link has been updated in the backend, then the appointment dashboard will show the Zoom link:

![calypso localhost_3000_me_concierge_sfg976597190 wordpress com_book (1)](https://user-images.githubusercontent.com/1269602/62029026-4aba2200-b1ff-11e9-9801-d760adc9fe6f.png)

#### Testing instructions



* Apply patches D30911-code and D30912-code and follow the testing instructions described in D30911-code.
* Verify that the `Reschedule or cancel` button redirects to the appropriate cancel session page(with the right site slug in the URL). 
    - verify that the back button in the cancel page works, and takes you back to the appointment dashboard.

Fixes # https://github.com/Automattic/concierge-admin/issues/29
